### ISSUE TITLE
[1.19.2] Crash when opening gallery with Fast Workbench installed.

### ISSUE THREAD

**Issue by Gbergz** [LGatodu47/ScreenshotViewer#5]
Title ^

Crash when opening the gallery, looks to be an issue together with [Fast Workbench](https://www.curseforge.com/minecraft/mc-mods/fastworkbench) mod. ?

```
The game crashed whilst unexpected error
Error: java.lang.NullPointerException: Cannot invoke "net.minecraft.resources.ResourceLocation.m_135815_()" because "b.f_94223_" is null
```

Full Crash-report: https://gist.githubusercontent.com/Gbergz/a89e1f341421d0a60d6d1e930b1672d8/raw/75af204351174d7fcdc82fa2ba034a0e4e908074/gistfile1.txt

### **Versions:**
**Minecraft:** 1.19.2
**Forge:** 43.2.3
**Screenshot Viewer:** v1.1.1
**Fast Workbench:** 7.0.1

**-** Gbergz

**Comment by LGatodu47** [comment]
It seems that the file on the curseforge page is not up to date with the github repository. Sorry for the inconvenience, a new version of the mod which should fix this for forge on 1.19.2 should be published. Anyway thanks for submitting this issue, let me know if it still persists.

**Comment by Gbergz** [comment]
Works now! Thanks :)
